src/main/java/mekanism/common/item/gear/ItemJetpack.java

```diff
@@ -128,6 +130,30 @@ public int getDefaultTooltipHideFlags(@Nonnull ItemStack stack) {
         return super.getDefaultTooltipHideFlags(stack);
     }
 
+    /**
+     * Gets the first found jetpack from an entity, if one is worn.
+     * <br>
+     * If Curios is loaded, the curio slots will be checked as well.
+     *
+     * @param entity the entity on which to look for the jetpack
+     *
+     * @return the jetpack stack if present, otherwise an empty stack
+     */
+    @Nonnull
+    public static ItemStack getJetpack(LivingEntity entity) {
+        return getJetpack(entity, entity.getItemBySlot(EquipmentSlot.CHEST));
+    }
+
+    @Nonnull
+    public static ItemStack getJetpack(LivingEntity entity, ItemStack chest) {
+        if (chest.getItem() instanceof ItemJetpack) {
+            return chest;
+        } else if (Mekanism.hooks.CuriosLoaded) {
```

Comment:
We're missing a case here for the mekasuit. I was messing around with something with the jetpack which I was thinking about making a pr for. But I'm not actually sure it's something which will get approved. Do you want me to put that in with it, and potentially refactoring the jetpack logic to an interface?

Reply:
Having an interface might be useful but the big thing about jetpack vs mekasuit (which is why the places that call this also special case handle checking the mekasuit (which the mekasuit jetpack doesn't work in the curio slots by design) is that the mekasuit is conditional on whether the module is even enabled.

Reply:
yeah, I only noticed this as it's not working currently, adding an interface is turning into a small scale refactor so it might just be faster to hotfix this and rebase my changes on that.